data/meson.build

```
@@ -0,0 +1,47 @@
+man_cfg=configuration_data()
+man_cfg.set('sysconfdir',sysconfdir)
+blocaled_8=configure_file(
+  input:'blocaled.8.in',
+  output:'blocaled.8',
+  configuration:man_cfg
+)
+install_man(blocaled_8)
+
+conf_cfg=configuration_data()
+conf_cfg.set('localeconfig',localeconfig)
+conf_cfg.set('keyboardconfig',keyboardconfig)
+conf_cfg.set('xkbdconfig',xkbdconfig)
+blocaled_conf = configure_file(
+  input: 'blocaled.conf.in',
+  output: 'blocaled.conf',
+  configuration:conf_cfg
+)
+install_data(blocaled_conf, install_dir: sysconfdir)
+
+serv_cfg=configuration_data()
+serv_cfg.set('libexecdir', prefix + '/' + get_option('libexecdir'))
```

```suggestion
serv_cfg.set('libexecdir', prefix / get_option('libexecdir'))
```

The `a / b` operator is syntactic sugar for `join_paths(a, b)`, it guarantees "smart" path joining regardless of edge cases such as "libexecdir lies outside of prefix so it ends up being an absolute path" (may be relevant on NixOS??) and is also just more readable if you ask me. :)